### DISCUSSION TITLE
Can I configure Scramble to use a different UI library?

### DISCUSSION THREAD

**Discussion by nerg4l** [dedoc/scramble#97] [Q&A] [answered]
I use Swagger UI in almost all projects. This is the first time I use Scramble which uses Stoplight Elements instead. Would it be possible to configure the library used for exposing a UI? It's not a deal breaker but would be useful in my opinion.

**Comment by romalytvynenko** [comment, marked as answer]
Hey @nerg4l 

Essentially you'd need some UI lib that supports Open API v3.1. Any will do.

The OpenAPI spec itself is available at `/docs/api.json`. So you'd simply need to give content at this URL to the lib.

Some coding may be needed depending on capabilities of the desired lib, but it looks possible to me!

Thanks!
